### PR TITLE
writing definition for LISTv2

### PR DESCRIPTION
**Pull request by nfarah86**
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/289)

### CODE REVIEW

**Comment by nfarah86** [comment]
@jtolds do you want to take a look at this for the listv2 def?

File: storage/boltdb/client.go [hunk 1]
```diff
@@ -145,6 +145,12 @@ func prevOrNext(reverseList bool, cur *bolt.Cursor) func() ([]byte, []byte) {
 	return cur.Next
 }
 
+//ListV2 is the new definition and will replace `List` definition
+func (c *Client) ListV2(opts storage.ListOptions) (storage.Items, storage.More, error) {
+	//TODO write the implementation
+	return storage.Items{}, false, nil
```
Comment (jtolds) [inline storage/boltdb/client.go]: my preference would be to panic with a TODO here

File: storage/redis/client.go [hunk 2]
```diff
@@ -112,6 +112,12 @@ func (c *Client) List(startingKey storage.Key, limit storage.Limit) (storage.Key
 	return listKeys, nil
 }
 
+//ListV2 is the new definition and will replace `List` definition
+func (c *Client) ListV2(opts storage.ListOptions) (storage.Items, storage.More, error) {
+	//TODO write the implementation
+	return storage.Items{}, false, nil
```
Comment (jtolds) [inline storage/redis/client.go]: same

**Comment by nfarah86** [comment]
merging

**Pull request merged**